package.json

```diff
@@ -25,30 +25,38 @@
     "Richard Gibson <richard.gibson@gmail.com>"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
-    "unist-util-visit-parents": "^2.0.0"
+    "@types/unist": "^2.0.0",
+    "unist-util-is": "^4.0.0",
+    "unist-util-visit-parents": "github:syntax-tree/unist-util-visit-parents#61d3786"
```

Comment:
TODO: replace this once `unist-utils-visit-parents` has it's next major release.